blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BluePipelineNode.java

```diff
@@ -79,6 +80,12 @@
     @Exported
     public abstract boolean isRestartable();
 
+    /**
+     * @return the id of the first parent
+     */
+    @Exported
```

Comment:
@olamy I missed this. ‘getFirstParent()’ is ambiguous. Is it first immediate parent or first node in the DAG?